lib/matplotlib/backends/_backend_tk.py

```diff
@@ -507,16 +507,22 @@ class NavigationToolbar2Tk(NavigationToolbar2, tk.Frame):
     Attributes
     ----------
     canvas : `FigureCanvas`
-        the figure canvas on which to operate
+        The figure canvas on which to operate.
     win : tk.Window
-        the tk.Window which owns this toolbar
-
+        The tk.Window which owns this toolbar.
+    pack_toolbar : bool, optional
```

Comment:
```suggestion
    pack_toolbar : bool, default: True
```
We're now using this shortend version, when possible.